core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1731,46 +1730,6 @@ private CompiledQuery createTableWithRetries(
         return executeWithRetries(createTableMethod, executionModel, configuration.getCreateAsSelectRetryCount(), executionContext);
     }
 
-    private CompiledQuery dropTable(SqlExecutionContext executionContext) throws SqlException {
-        // expected syntax: DROP TABLE [ IF EXISTS ] name [;]
-        CharSequence tok = SqlUtil.fetchNext(lexer);
-        if (tok == null || !SqlKeywords.isTableKeyword(tok)) {
-            return unknownDropStatement(executionContext, tok);
```

Comment:
This change broke enterprise version as both the `unknownDropStatement` and `unknownDropTableSuffix` methods were used there. @marregui could you check `DropTableTest` there and fix it?

Reply:
I have created https://github.com/questdb/questdb-enterprise/pull/70 with a propossed fix